netci.groovy

```diff
@@ -34,6 +34,9 @@ platformList.each { platform ->
     else if (os == 'Ubuntu') {
         buildCommand = "./build.sh --skip-prereqs --configuration ${configuration} --docker ubuntu --targets Default"
     }
+    else if (os == 'Ubuntu16.04') {
```

Comment:
I think you actually need to remove this.  I don't think the Ubuntu 16.04 machines have docker installed.